<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimerDialog</class>
 <widget class="QWidget" name="TimerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>215</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Description:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="8">
    <widget class="QLineEdit" name="lineEditComment">
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Player:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="7">
    <widget class="QLineEdit" name="lineEditPlayer">
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="9">
    <widget class="QPushButton" name="pushButtonPlayer">
     <property name="text">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Sound file:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="7">
    <widget class="QLineEdit" name="lineEditSoundFile">
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="9">
    <widget class="QPushButton" name="pushButtonSoundFile">
     <property name="text">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Run at</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="2">
    <widget class="QSpinBox" name="spinBoxHours">
     <property name="suffix">
      <string> hr</string>
     </property>
     <property name="prefix">
      <string/>
     </property>
     <property name="maximum">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item row="3" column="4" colspan="2">
    <widget class="QSpinBox" name="spinBoxMinutes">
     <property name="suffix">
      <string> min</string>
     </property>
     <property name="prefix">
      <string/>
     </property>
     <property name="maximum">
      <number>59</number>
     </property>
    </widget>
   </item>
   <item row="3" column="6" colspan="2">
    <widget class="QPushButton" name="pushButtonCurrent">
     <property name="text">
      <string>Now</string>
     </property>
    </widget>
   </item>
   <item row="3" column="8" colspan="2">
    <widget class="QPushButton" name="pushButtonReset">
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="5">
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>Show OSD notification</string>
     </property>
    </widget>
   </item>
   <item row="4" column="5" colspan="5">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="10" alignment="Qt::AlignTop">
    <widget class="QLabel" name="errorLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
